mobot/mobot_client/drop_session.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+
+import enum
+
+import full_service_cli as mc
+from django.utils import timezone
+from mobot_client.models import DropSession, Drop, CustomerStorePreferences, Order, Sku
+
+from mobot_client.chat_strings import ChatStrings
+
+
+class SessionStateReadyToReceiveInitial(enum.Enum):
+    NOT_READY = 0
+    READY = 1
+
+
+class SessionState(enum.Enum):
+    CANCELLED = -1
+    READY_TO_RECEIVE_INITIAL = 0
+    WAITING_FOR_BONUS_TRANSACTION = 1
+    ALLOW_CONTACT_REQUESTED = 2
+    COMPLETED = 3
+
+
+class ItemSessionState(enum.Enum):
+    IDLE_AND_REFUNDABLE = -4
+    IDLE = -3
+    REFUNDED = -2
+    CANCELLED = -1
+    NEW = 0
+    WAITING_FOR_PAYMENT = 1
+    WAITING_FOR_SIZE = 2
+    WAITING_FOR_NAME = 3
+    WAITING_FOR_ADDRESS = 4
+    SHIPPING_INFO_CONFIRMATION = 5
+    ALLOW_CONTACT_REQUESTED = 6
+    COMPLETED = 7
+
+    @classmethod
+    def active_states(cls):
+        return {
+                cls.NEW.value,
+                cls.WAITING_FOR_PAYMENT.value,
+                cls.WAITING_FOR_SIZE.value,
+                cls.WAITING_FOR_NAME.value,
+                cls.WAITING_FOR_ADDRESS.value,
+                cls.SHIPPING_INFO_CONFIRMATION.value,
+                cls.ALLOW_CONTACT_REQUESTED.value
+            }
+
+    @classmethod
+    def refundable_states(cls):
+        return {
+            cls.IDLE_AND_REFUNDABLE.value,
+            cls.WAITING_FOR_SIZE.value,
+            cls.WAITING_FOR_ADDRESS.value,
+            cls.WAITING_FOR_ADDRESS.value,
+            cls.WAITING_FOR_NAME.value,
+            cls.SHIPPING_INFO_CONFIRMATION.value,
+        }
+
+
+class DropType(enum.Enum):
```

Comment:
Same as @eranrund suggested, can we convert these to models.IntegerChoices? We can just do `from django.db import models` at the top, and switch out the `enum.Enum` to `models.IntegerChoices`.

Reply:
Done